src/metabase/models/revision.clj

```diff
@@ -73,7 +73,9 @@
 
 (t2/define-before-insert :model/Revision
   [revision]
-  (assoc revision :timestamp :%now))
+  (assoc revision
+         :timestamp :%now
+         :most_recent true))
```

Comment:
Suggestion: call this `last_edit`, which is consistent with `last-edit-info`.

Also, we'll probably want to use this in other places, like here: 
https://github.com/metabase/metabase/blob/c40185a8d036248875e6723210d29db136a5c1ab/src/metabase/models/revision/last_edit.clj#L36-L50

Reply:
I think I prefer `most_recent` better. This idea is from `moderation_review.most_recent` which has the same purpose.

> we'll probably want to use this in other places, like here:

yep, I planned to have a follow making use of this new flag because I'm afraid this PR went too big. but since you mentioned it I can update it here too.

Reply:
I think I prefer `most_recent` better. This idea is from `moderation_review.most_recent` which has the same purpose.

> we'll probably want to use this in other places, like here:

yep, I planned to have a follow making use of this new flag because this has PR has ~1000 changes already.